build/pxc-entrypoint.sh

```diff
@@ -454,6 +456,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			else
 				echo >&2 "WSREP: Failed to recover position: "
 				cat "$wsrep_verbose_logfile"
+				rm "$wsrep_verbose_logfile"
```

Comment:
why not move `rm` command to line 463

Reply:
we have `exit 1 `  line 460, so in this case  file will not be removed  at all